test/unit/detail/type_name_as_string_test.cpp

```diff
@@ -0,0 +1,53 @@
+// -----------------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/sharg-parser/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
```

Comment:
```suggestion

#include <type_traits>

```

for `std::is_same_v`